src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -473,7 +473,7 @@ private async Task<bool> AreCentralVersionRequirementsSatisfiedAsync(RestoreRequ
             // Log a warning if there are more than one configured source and package source mapping is not enabled
             if (restoreRequest.Project.RestoreMetadata.Sources.Count > 1 && !restoreRequest.PackageSourceMapping.IsEnabled)
             {
-                await _logger.LogAsync(RestoreLogMessage.CreateWarning(NuGetLogCode.NU1507, Strings.Warning_CentralPackageVersions_MultipleSourcesWithoutPackageSourceMapping));
+                await _logger.LogAsync(RestoreLogMessage.CreateWarning(NuGetLogCode.NU1507, string.Format(CultureInfo.CurrentCulture, Strings.Warning_CentralPackageVersions_MultipleSourcesWithoutPackageSourceMapping, restoreRequest.Project.RestoreMetadata.Sources.Count)));
```

Comment:
Reading the linked issue, I wonder if this message should include the names/urls of the package sources as well.

Reply:
I had started this change and PR before I saw Jimmy's issue with the additional concerns about the message. I went ahead and just sent it just correcting the bug since I wasn't sure what your team would want in the message or not. But I'm happy to update the PR with more detail if you all want to provide different language.

Reply:
For now we're going to just print the number of sources so we can get it fixed in the next release.  Then we'll print the sources in a follow-up change.

Reply:
> For now we're going to just print the number of sources so we can get it fixed in the next release. Then we'll print the sources in a follow-up change.

Can you please make sure the issue is created. @jeffkl